components/combobox/combobox.jsx

```diff
@@ -382,6 +412,69 @@ class Combobox extends React.Component {
 		}
 	}
 
+	getCustomPopoverProps = (body, { assistiveText }) => {
+		/*
+		 * Generate the popover props based on passed in popover props. Using the default behavior if not provided by passed in popover
+		 */
+		const popoverBody = (
+			<div>
+				<div className="slds-assistive-text" id={`${this.getId()}-label`}>
+					{assistiveText.popoverLabel}
+				</div>
+				{body}
+			</div>
+		);
+
+		const popoverFooter = (
+			<div>
+				<button
+					className="slds-button slds-button_neutral"
+					id="cancel-button"
+					onClick={(e) => {
+						// onClose(e, { trigger: 'cancel' });
+						this.handleClose(e, { trigger: 'cancel' });
+					}}
+				>
+					Cancel
+				</button>
+				<button
+					className="slds-button slds-button_brand"
+					ref={this.lastTab}
+					id="done-button"
```

Comment:
`done-button` is fine for a suffix, but should use the combobox `getId` so that it is always unique on the page.